### DISCUSSION TITLE
Filmbox - ALF-4   logc4->Logc4 Lut support

### DISCUSSION THREAD

**Discussion by finnschi** [videovillage/community#152] [Ideas]
I would like to easily generate logc4-> logc4 luts for the alexa35 alf-4 workflow . 

I currently have a construct of colorspace transforms to archieve a lut that I can use in-camera and as a shotgrade, would make it much easier for less colorscience inclined DPs to be able to generate those luts.

**Comment by gregcotten** [comment]
There's actually no technically correct way of doing this yet, unfortunately!

**Comment by LaurentRipoll** [comment]
Hello,
Is there any news on this subject? I desperately need to output filmbox LUTs to LogC4, and it's impossible to output LUTs working in ACES to LOGC4 even using Color Space Transform, anyone have any ideas?

**Reply by gregcotten** [reply]
ARRI will be announcing something VERY relevant to this issue shortly. Unfortunately that's all I'm able to disclose at the moment.
